pkg/errors/reconcile_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Upbound Inc.
+// All rights reserved
```

Comment:
Should be the Crossplane copyright stuff here.

Reply:
indeed

Reply:
fixed